lib/chef/knife/azurerm_base.rb

```diff
@@ -27,6 +27,10 @@ class Chef
   class Knife
     module AzurermBase
 
+      ## azure-xplat-cli versio that introduced deprecation of Windows Credentials
+      ## Manager (WCM) usage for authentication credentials storage purpose ##
+      XPLAT_VERSION_WITH_WCM_DEPRECATED ||= "0.10.5"
```

Comment:
@btm I have used `||=` to avoid warning like `warning: already initialized constant` and `warning: previous definition of ..`. Though the constant is declared at class level, still these warning are raised.